src/Lykke.Service.PayInternal.Core/Domain/Asset/AssetAvailabilityByMerchant.cs

```diff
@@ -1,13 +1,11 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Lykke.Service.PayInternal.Core.Domain.Asset
+﻿namespace Lykke.Service.PayInternal.Core.Domain.Asset
 {
     public class AssetAvailabilityByMerchant : IAssetAvailabilityByMerchant
     {
         public string MerchantId { get; set; }
-        public string AssetsPayment { get; set; }
-        public string AssetsSettlement { get; set; }
+
+        public string PaymentAssets { get; set; }
```

Comment:
It would be better to use IReadOnlyList in domain model. It does not matter how does it stored in DB.

Reply:
The point is client already implemented and expects value as single string with separators. So, there is no reason to convert string to IReadonlyList after reading from DB and convert it back to string before sending the response to client.